applications/nrf_desktop/configuration/nrf52840_pca10059/app_ZReleaseMCUBoot.conf

```diff
@@ -71,6 +71,11 @@ CONFIG_SYS_PM_POLICY_APP=y
 
 CONFIG_GPIO=y
 
+CONFIG_SERIAL=n
+CONFIG_UART_0_NRF_UARTE=n
+CONFIG_UART_INTERRUPT_DRIVEN=n
+CONFIG_CONSOLE=n
```

Comment:
Disable `USE_SEGGER_RTT` ?

Reply:
board config does not enable it so we don't have to